azure/agent.pkr.hcl

```diff
@@ -42,46 +39,58 @@ variable "openssh_version" {
   type = string
 }
 variable "client_id" {
-  type = string
+  type    = string
+  default = ""
 }
 variable "client_secret" {
-  type = string
+  type    = string
+  default = ""
 }
 variable "subscription_id" {
-  type = string
+  default = ""
+  type    = string
+}
+variable "image_version" {
+  type    = string
+  default = "0.0.2"
```

Comment:
do we have to bump this if we update the image? what's the workflow?

Reply:
- For this PR: no need to bump: rebuilding the image will override the existing image in version `0.0.2`, same a current behavior. It's only required to allow using collection.
- Plan: upcoming PR to use `jx-next-version` to determine and bump this value (and add automatic tagging)

WDYT?

Reply:
both work thanks